django_airavata/static/common/scss/main.scss

```diff
@@ -136,7 +136,7 @@ body {
 .card {
     background-color: #fff;
     border-radius: 4px;
-    border: none;
+    border: 1px solid rgba(0,0,0,.125);
```

Comment:
Can you undo this change? If you need a border around the card, just add the `border-default` class, see line 143.